### ISSUE TITLE
Nabla - price of AMPE is not correct

### ISSUE THREAD

**Issue by annatekl** [pendulum-chain/portal#532]
### Describe the bug

The price of AMPE token is not correct

### Reproduction

1. Go to Nabla Swap
2. Click on Swap
3. Choose AMPE and any asset
4. See the price for AMPE = 0.1 $

### Expected behavior

The price should match the correct AMPE price from Zenlink, which is 0.002 as of July 26th

### Screenshots

<img width="776" alt="Screenshot 2024-07-26 at 13 30 07" src="https://github.com/user-attachments/assets/552bcbd4-343f-4533-95e3-dce474f1a6ed">

**Comment by ebma** [comment]
On which network is this? Isn't this Foucoco anyway?

**Comment by prayagd** [comment]
yes this is foucoco, moved down the backlog. Again sorry for the ping, are pings needed for bug tickets are should we just put them in top if critical and that would be enough for you to get eyes on it?

**Comment by ebma** [comment]
Moving them to the top of the backlog should be enough as long as you don't require any immediate additional input for it.
